src/background/background.ts

```diff
@@ -1,4 +1,151 @@
+Implemented a user-friendly interface for clients to schedule appointments and an admin panel for managing schedules.
+	Github Repository: Client, Server
+	Deployment: https://planifyapp.netlify.app/
+
+ChatGPT for Audio-Jungle - Chrome Extension
+April 2023 - July 2023
+Developed a Chrome extension for sound designers, to assist users with sound effects submission within stock sound effects platforms.
+Utilizing ChatGPT API for enhanced sound effect library selling experience.
+Leveraged React for the user interface and integrated with ChatGPT for natural language processing.
+EDUCATION
+BSc. in Computer Science, Sorbonne Université - Campus Pierre et Marie Curie— Paris
+2012 - 2015
+Diploma in Audio Engineering, SAE Institute — Paris
+2015 - 2017
+Online courses
+Ongoing
+Continuously enhancing skills through Udemy courses, including:
+Beginning C++ Programming - From Beginner to Beyond
+The Complete JavaScript Course
+React - The Complete Guide
+Node.js, Express, MongoDB & More
+The Ultimate MySQL Bootcamp: Go from SQL Beginner to Expert
+React & TypeScript Chrome Extension Development
+JavaScript Algorithms and Data Structures Masterclass
+Software Architecture & Design of Modern Large Scale Systems
+PREVIOUS EXPERIENCE
+
+Self Employed, Tel Aviv — Sound Designer
+March 2017 - January 2023
+As the ex-owner of a top-notch sound studio in Israel, based in Tel Aviv, I led the way on audio design projects for big video production companies. Working with huge names like McDonald's, AMPM, Samsung, Strauss, Tnuva, and providing sound design solutions for big tech companies such as Wix, SAP, Fiverr, Tabula, Simply, Viber, and others. I consistently delivered top-quality work.
+Successfully sold the business for a profit.
+
+
+SKILLS
+Deep understanding of Vanilla Javascript
+fluency with relational and NoSQL databases(MySQL, MongoDB)
+Proficient in implementing best practices for designing and maintaining APIs.
+Experienced in applying responsive design principles for optimal user experience.
+Experience with Git and GitHub
+Experienced in Chrome extension development
+Experienced in Vs Code extension development
+PROGRAMING LANGUAGES
+Javascript, Typescript, C++
+FRAMEWORKS
+Express, ReactJS
+`;
+// Global states - Chrome Storage ---------------------
+
+type GlobalStateKey = 'query'; // we might need:  'jobDescription' | 'currentUrl'  | 'jobPostId';
+
+function setState(key: GlobalStateKey, value: any) {
+  chrome.storage.sync.set({ [key]: value }, () => {
+    console.log(`Data saved - key: ${key} and value: ${value}`);
+  });
+}
+
+function getState(key: GlobalStateKey) {
+  return new Promise((resolve, reject) => {
+    chrome.storage.sync.get(key, (result) => {
+      if (chrome.runtime.lastError) {
+        reject(chrome.runtime.lastError);
+      } else {
+        resolve(result[key]);
+      }
+    });
+  });
+}
+
+function sendToContentScript(message: Message) {
```

Comment:
this is a useful util function, better place it in a utils file so it can be used from different places

Reply:
Done, I thought it also makes sense to move the other functions to utils in that case.